setup.py

```diff
@@ -0,0 +1,7 @@
+import setuptools
+
+setuptools.setup(
+    name="converter",
+    version="0.0.1",
+    packages=setuptools.find_packages('.'),
+)
```

Comment:
Please add the following information: "Description", "Author".
Also, the name "convert" is too generic.

Reply:
Also add minimum python version, or at least the version this code was tested with.